src/buildercore/fastly/vcl/error-page.vcl.tpl

```diff
@@ -0,0 +1,5 @@
+if (obj.status == ${code}) {
+   set obj.http.Content-Type = "text/html; charset=us-ascii";
```

Comment:
Doesn't look like Terraform provides response headers, so we have to set this somewhere.

Reply:
`header` is available but depends on Fastly's request/cache/response conditions so it's not as flexible